docker-compose.yml

```diff
@@ -1,13 +1,12 @@
 version: '3'
 
 services:
-  artemis-app:
-    build:
-      context: .
-      dockerfile: ./docker/app/Dockerfile
-    command: ./gradlew bootRun
+  artemis-server:
+    command: sh -c "./gradlew build -xwebpackBuildDev -xwebpack -xtest
```

Comment:
I think we should try to move this command to our build.gradle, otherwise it might become outdated.
I would personally make the boot with server & client the special case and the normal bootRun only start the server (when you do proper development you have to start both individually).

Reply:
You're talking about the gradle `build` task including the skipped tasks, right? I think `build` is the most common gradle task, but I don't mind to put it in a separate task.

Don't know if I get you right on your second point, but your suggestion is to make a general change to the `bootRun` gradle task (server+client -> only server)? If so, I highly agree but this might be a little bit out of scope for this pull request and it doesn't seem to be that easy either, because even if you skip `webpackBuildDev` and `webpack` tasks, the frontend is still partially shipped on the default port.

Reply:
Resolved via ccc40abc4311a16db06c90a3d7362e8c7bba7065